packages/daimo-api/src/router.ts

```diff
@@ -113,9 +113,31 @@ export function createRouter(
           throw new Error(`Invalid Daimo app link: ${url}`);
         }
 
+        async function getEAccountFromLinkString(
+          account: string
+        ): Promise<EAccount> {
```

Comment:
if you want this to not be trial/error based, one option:

```
if (eAccStr.includes(".")) {
 // parse as ENS
} else if (isAddress(eAccStr)) {
  // return getAddress
} else {
  // daimo name
}

Reply:
done